spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -202,7 +202,7 @@
           :virtv2v_started_on => 1
         }
         task.conversion_host = conversion_host
-        allow(conversion_host).to receive(:get_conversion_state).with(task.options[:virtv2v_wrapper]['state_file']).and_return({})
+        allow(task).to receive(:get_conversion_state).and_return([])
```

Comment:
`Task#get_conversion_state` now returns an Array instead of a Hash?

Reply:
Do we need to stub this method call?

Reply:
looks like it still returns a hash

Reply:
This was added by someone else.  Tests pass even when "allow(task).to ..." statement is taken out.  Currently investigating the reason for this statement to be here.